tests/Application/config/sylius/1.8/bundles.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
```

Comment:
Is this still required if minimal Sylius version is 1.9 specified in composer.json?